### ISSUE TITLE
Getting errors calling getFeed()

### ISSUE THREAD

**Issue by scebbers** [pgrimaud/instagram-user-feed#55]
Initializing the Api runs without errors, but when calling getFeed() on the api object I get the following warnings, and the feed seems empty.

```
$cache = new Instagram\Storage\CacheManager('instacache');
$api = new Instagram\Api($cache);
$api->setUserName('garagezuidbroek');
$feed = $api->getFeed();
```
The list of notices is long, but the first three are:
```
Notice: Undefined property: stdClass::$ProfilePage in /customers/2/5/3/***.***/httpd.www/m/includes/vendor/pgrimaud/instagram-user-feed/src/Instagram/Transport/HtmlTransportFeed.php on line 60

Notice: Trying to get property 'graphql' of non-object in /customers/2/5/3/***.***/httpd.www/m/includes/vendor/pgrimaud/instagram-user-feed/src/Instagram/Transport/HtmlTransportFeed.php on line 60

Notice: Trying to get property 'user' of non-object in /customers/2/5/3/***.***/httpd.www/m/includes/vendor/pgrimaud/instagram-user-feed/src/Instagram/Transport/HtmlTransportFeed.php on line 60
```

This is when deploying to shared hosting (one.com), on my own server this exact same code works fine.

**Comment by pgrimaud** [comment]
Hello,

If you are on shared hosting, could you try with login like this example : https://github.com/pgrimaud/instagram-user-feed/blob/master/examples/getMediasWithLogin.php ?

**Comment by scebbers** [comment]
There seems to be no function login on the Api class in my version (5.5.3).
`Fatal error: Uncaught Error: Call to undefined method Instagram\Api::login() in /customers/2/5/3/***.***/httpd.www/m/includes/instafeed.inc.php:19`

**Comment by scebbers** [comment]
Using version 5.6-beta I receive the following response in GuzzleHttp:

```
Fatal error: Uncaught GuzzleHttp\Exception\ClientException: Client error: `POST https://www.instagram.com/accounts/login/ajax/` resulted in a `400 Bad Request` response:
{"message": "checkpoint_required", "checkpoint_url": "/challenge/34250056/E8dV4LU6KT/", "lock": false, "status": "fail"}
 in /customers/2/5/3/***.***/httpd.www/m/includes/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:113
```

**Comment by martinsnajdr** [comment]
I have the same problem since I think yesterday. Suddenly getFeed stopped working on our server, but still working on my localhost.

**Comment by scebbers** [comment]
I now basically have the same problem as in issue #48. Instagram does not let my remote server login without email verification, and I cannot find a way around it. It also blocked the IP of my server.
I have turned to version 4.0 for now to retrieve media until a solution is found.

**Comment by maksimpalii** [comment]
Undefined property: stdClass::$ProfilePage
http://prntscr.com/qy5bnr

**Comment by Friend686** [comment]
"Undefined property: stdClass::$ProfilePage" - same problem.

**Comment by pgrimaud** [comment]
Hello, I released a new version (6.0). Could you upgrade to this one?
